test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -532,5 +533,68 @@ describe('View existing record,', function() {
                 
             }); 
         });
+        
+        describe("For a related entity wuth an association table and markdown display", function () {
+            beforeAll(function() {
+                var keys = [];
+                keys.push(testParams.key.name + testParams.key.operator + testParams.key.value);
+                browser.ignoreSynchronization=true;
+                var url = browser.params.url + "/record/#" + browser.params.catalogId + "/" + testParams.schemaName + ":" + testParams.table_name + "/" + keys.join("&");
+                browser.get(url);
+                chaisePage.waitForElement(element(by.id('rt-heading-association_table_markdown')));
+            });
+            
+            it("on adding new relationship should update the table display", function (){
```

Comment:
Since you have multiple `expect` statements inside this `it`, can you please add error message to each of them?